iib/workers/tasks/opm_operations.py

```diff
@@ -193,6 +193,130 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
 
 
+def _get_temp_index_db_file(base_dir, from_index):
```

Comment:
What is the use of the function? Could we just get the file name from the "main" function  `with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:`

Reply:
it is not getting name but making sure that the file exist on returned path